src/core/qgstranslationcontext.h

```diff
@@ -0,0 +1,95 @@
+/***************************************************************************
+  qgstranslationcontext.h
+
+ ---------------------
+ begin                : 23.5.2018
+ copyright            : (C) 2018 by David Signer
+ email                : david at opengis dot ch
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+#ifndef QGSTRANSLATIONCONTEXT_H
+#define QGSTRANSLATIONCONTEXT_H
+
+#include "qgis_core.h"
+#include "qgis_sip.h"
+#include "qgis.h"
+
+class QgsProject;
+
+/**
+ * \ingroup core
+ * \class QgsTranslationContext
+ * \brief used for the collecting of strings of .qgs to be translated and writing of ts file
```

Comment:
Capitalization

"Used for collecting strings from projects for translation and creation of ts files."